src/_common/content/components/media-item/media-item.ts

```diff
@@ -130,6 +132,12 @@ export default class AppContentMediaItem extends Vue {
 		return !this.imageLoaded && !GJ_IS_SSR;
 	}
 
+	get useResponsiveImg() {
```

Comment:
shouldUseReponsiveImg is the convention we use

Reply:
Also, I actually think this should be changed to `shouldUseMediaserver` since we're using the img-responsive class on both flows. It's more that we want to do the mediaserver flow in certain cases.